docs/content/releases.md

```diff
@@ -6,6 +6,39 @@ doctypes: ["concept"]
 toc: true
 docs: "DOCS-616"
 ---
+## 3.5.2
+
+31 May 2024
+
+### <i class="fa-solid fa-triangle-exclamation"></i> Important
+
+- Bundles compiled on NAP WAF versions <= v4.8.x are not compatible with NAP WAF versions >= 4.9.x, this release of NIC includes NAP WAF v4.10 so recompilation of policy bundles is required.
```

Comment:
My only question is if this might create confusion for customers using JSON based Policy.  Which we believe is the majority.  "Policy bundles" is clear to us, but I suspect is not clear to most of our WAF customers.
Just noting this, but not blocking.